src/components/molecules/Popover/Popover.stories.tsx

```diff
@@ -29,24 +30,37 @@ const positions: PopoverProps["position"][] = [
 
 const SampleContent = () => (
   <div className="rounded-lg bg-white p-2 shadow-04">
-    <div className="p-2">Text</div>
-    <div className="p-2">Text</div>
+    <span>Content</span>
   </div>
 );
 
-export const Default: StoryObj<typeof Popover> = {
-  args: {
-    children: (
+const WithStateComponent = (props: PopoverProps) => {
+  const [isOpen, setIsOpen] = useState(props.isOpen);
+
+  return (
+    <Popover
+      {...props}
+      isOpen={isOpen}
+    >
       <Button
         theme="primary"
         size="small"
+        onClick={() => props.mode === "click" && setIsOpen(!isOpen)}
```

Comment:
setIsOpen(prev => !prev)
```suggestion
        onClick={() => props.mode === "click" && setIsOpen(prev => !prev)}
```